scripts/build-only-wasm.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
```

Comment:
should node-template/scripts have this script as well?

Reply:
Always use `/usr/bin/env`. Posix even recommends that. While Nixos also has `/bin/sh` to be Posix compliant and `/usr/bin/env` to support scripts, `/usr/bin/env sh` is the cleaneast way.

https://news.ycombinator.com/item?id=17069408